src/power_grid_model/data_types.py

```diff
@@ -11,7 +11,7 @@
 
 # Import non class symbols as private to reassign and provide a docstring.
 # Not necessary for class, as sphinx autodoc finds their docstrings
-from power_grid_model._core.data_types import (  # pylint: disable=unused-import
+from power_grid_model._core.data_types import (  # noqa: F401, I001
```

Comment:
i prefer disabling by message, not by line. does `ruff` not support that? idem below

EDIT: nope, ruff only supports disabling codes